pkg/hintrunner/zero/zerohint_blake_test.go

```diff
@@ -521,5 +517,53 @@ func TestZeroHintBlake(t *testing.T) {
 				errCheck: errorTextContains(fmt.Sprintf("in range [0, 100), got %d", 200)),
 			},
 		},
+		"Blake2sCompute": {
+			{
+				operanders: []*hintOperander{
+					{Name: "output", Kind: apRelative, Value: addrWithSegment(1, 31)},
```

Comment:
you can just do  `addr(..)` as it resolves to `vm.ExecutionSegment` which is `1` 
but its clearer like this also i agree !